src/ports/postgres/modules/pmml/test/pmml.setup.sql_in

```diff
@@ -0,0 +1,363 @@
+/* ----------------------------------------------------------------------- *//**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ *//* ----------------------------------------------------------------------- */
+
+ DROP TABLE IF EXISTS houses CASCADE;
```

Comment:
Can we move the dataset outside of this setup file? This is mainly to create tables relevant to the test rather than the entire pmml testsuite.

Reply:
I think I'll take care of this in the next PR since that PR will also touch some other pmml modules and it will easier to refactor all the datasets together